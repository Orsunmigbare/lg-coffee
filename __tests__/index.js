import React from "react";
import renderer from "react-test-renderer";
import Index from "../pages/index";

//needs to fetch data first

it("renders homepage unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
