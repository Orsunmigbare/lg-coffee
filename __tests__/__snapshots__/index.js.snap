// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg sticky-top"
    style={
      Object {
        "background": "#FAFAFA",
      }
    }
  >
    <a
      className="navbar-brand text-light"
      href="/"
    >
      <img
        alt=""
        className="d-inline-block align-top"
        height="30"
        loading="lazy"
        src="/logo.png"
        width="30"
      />
    </a>
    <button
      aria-controls="navbarNav"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarNav"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNav"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="undefined nav-item"
        >
          <a
            className="nav-link"
            href="/"
            style={
              Object {
                "color": "#181818",
              }
            }
          >
            Products
             
            <span
              className="sr-only"
            >
               
            </span>
          </a>
        </li>
        <li
          className="undefined nav-item"
        >
          <a
            className="nav-link"
            href="/made-to-order"
            style={
              Object {
                "color": "#181818",
              }
            }
          >
            Made To Order
             
            <span
              className="sr-only"
            >
               
            </span>
          </a>
        </li>
        <li
          className="undefined nav-item"
        >
          <a
            className="nav-link"
            href="/about"
            style={
              Object {
                "color": "#181818",
              }
            }
          >
            About
             
            <span
              className="sr-only"
            >
               
            </span>
          </a>
        </li>
      </ul>
    </div>
  </nav>,
  <div>
    <div
      className="fluid-container hero_image"
      style={
        Object {
          "backgroundImage": "url(\\"/heatpress.jpeg\\")",
        }
      }
    >
      <div
        className=""
      >
        <h1
          className="font-weight-bold"
        >
          Made To Order
        </h1>
        <p
          className="font-weight-normal"
        >
          The highest quality work comes from custom work.
        </p>
        <a
          className="btn btn-outline-dark"
          href="/products"
        >
          Start Your Order
        </a>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="d-flex products_wrapper"
      >
        <section
          className="product_card_v m-1"
        >
          <a
            className="no-link"
            href="/view-product/bun_vs_"
          >
            <div
              className="product_image"
              style={
                Object {
                  "backgroundImage": "url(\\"/front_page_bunns@2x.png\\")",
                }
              }
            >
              <h2
                className="font-weight-bold"
              >
                Bunns In The Void
              </h2>
            </div>
          </a>
        </section>
        <section
          className="product_card_v m-1"
        >
          <a
            className="no-link"
            href="/view-product/lad_vs_"
          >
            <div
              className="product_image"
              style={
                Object {
                  "backgroundImage": "url(\\"/front_page_lady@2x.png\\")",
                }
              }
            >
              <h2
                className="text-light font-weight-bold"
              >
                Lady In The Void
              </h2>
            </div>
          </a>
        </section>
      </div>
    </div>
  </div>,
  <div
    className="container mt-5 mb-4 dark_mode"
  >
    <ul
      className="nav justify-content-center"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-light"
          href="#"
        >
          Contact
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-light"
          href="#"
        >
          Collections
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-light"
          href="#"
        >
          FAQ
        </a>
      </li>
    </ul>
    <div
      className="d-flex justify-content-center"
    >
      <span
        className="text-light"
        style={
          Object {
            "fontSize": ".75em",
          }
        }
      >
        2020
      </span>
    </div>
  </div>,
]
`;
